spec/services/case_filter/open_case_status_filter_spec.rb

```diff
@@ -198,6 +213,40 @@
     end
   end
 
+  describe "#available_choices" do
+    context "when logged in as a Branston user" do
+      let(:user) { find_or_create :branston_user }
+      let(:open_case_status_filter) { described_class.new search_query, user, Case::Base }
+      let(:search_query) do
+        create :search_query
+      end
+
+      it "has the rejected key" do
+        expect(open_case_status_filter.available_choices[:filter_open_case_status]).to have_key("rejected")
+      end
+
+      it "returns the filter list with rejected as 'rejected'" do
+        expect(open_case_status_filter.available_choices[:filter_open_case_status]["rejected"]).to eq "Rejected"
+      end
+    end
+
+    context "when logged in as a London Disclosure user" do
+      let(:user) { find_or_create :disclosure_specialist_bmt }
+      let(:open_case_status_filter) { described_class.new search_query, user, Case::Base }
+      let(:search_query) do
+        create :search_query
+      end
+
+      it "has the rejected key" do
+        expect(open_case_status_filter.available_choices[:filter_open_case_status]).not_to have_key("rejected")
+      end
+
+      it "returns the filter list with rejected as nil" do
```

Comment:
Worth removing this now?

Reply:
I don't really understand the purpose of it, so probably :)

Reply:
I think you just need a test to check that there is no rejected key present